packages/appstore/src/modules/trading-hub/index.tsx

```diff
@@ -1,7 +1,21 @@
 import * as React from 'react';
+import { useStores } from 'Stores';
+import { Authorize } from '@deriv/api-types';
+import WalletAccount from 'Components/wallet-account';
+import './trading-hub.scss';
 
 const TradingHub = () => {
-    return <div className='trading-hub'>Trading Hub</div>;
+    const { client } = useStores();
+
+    const wallet_accounts = (client.wallet_accounts || []) as Required<Authorize>['account_list'];
+
+    return (
+        <div className='trading-hub'>
```

Comment:
@iman-fs If `wallet_accounts` empty, This `div` will still render which has `4rem` padding, Is this intentional?

Reply:
`<div className='trading-hub'></div>` is the container for trading-hub page. 
and wallet_accounts won't be empty because when users create an deriv account, a default demo wallet account will be created for them.